src/test/java/com/commercetools/sync/sdk2/products/utils/ProductSyncMockUtils.java

```diff
@@ -202,15 +237,76 @@ public static CategoryOrderHints createRandomCategoryOrderHints(
           categoryOrderHints.put(
               resourceIdentifier.getId(), ORDER_HINT_FORMAT.format(randomDouble));
         });
-    return CategoryOrderHints.of(categoryOrderHints);
+    return CategoryOrderHintsBuilder.of().values(categoryOrderHints).build();
+  }
+
+  public static ProductVariantDraft createProductVariantDraft(final ProductVariant productVariant) {
+    List<AssetDraft> assetDrafts = createAssetDraft(productVariant.getAssets());
+    List<PriceDraft> priceDrafts = createPriceDraft(productVariant.getPrices());
+    return ProductVariantDraftBuilder.of()
+                                     .assets(assetDrafts)
+                                     .attributes(productVariant.getAttributes())
+                                     .images(productVariant.getImages())
+                                     .prices(priceDrafts)
+                                     .sku(productVariant.getSku())
+                                     .key(productVariant.getKey())
+        .build();
+  }
+
+  public static List<AssetDraft> createAssetDraft(List<Asset> assets) {
+    return assets.stream()
+        .map(asset -> AssetDraftBuilder.of()
+                                       .custom(createCustomFieldsDraft(asset.getCustom()))
+                                       .description(asset.getDescription())
+                                       .name(asset.getName())
+                                       .key(asset.getKey())
+                                       .sources(asset.getSources())
+                                       .tags(asset.getTags()).build())
+        .collect(Collectors.toList());
+  }
```

Comment:
I would put this method into a new class `AssetUtils` which has separate role from this class

Reply:
[837a7a9](https://github.com/commercetools/commercetools-sync-java/pull/886/commits/837a7a9c3750b41c90c179a32dd6e69e1e7ad68a)